test/techs.e2e-spec.ts

```diff
@@ -215,6 +225,278 @@ describe("Techs Controller (e2e)", () => {
         });
     });
 
+    describe("PATCH voyages/teams/:teamId/techs - update tech stack item of the team", () => {
+        it("should return 200 and update a tech stack item", async () => {
+            const teamId: number = 2;
+            const techId: number = 1;
+            const teamMemberId: number = 8;
+
+            return request(app.getHttpServer())
+                .patch(`/voyages/teams/${teamId}/techs`)
+                .set("Authorization", `Bearer ${userAccessToken}`)
```

Comment:
please use cookie instead of bearer as we have moved from bearer token to http cookie and would like to standardize the tests, you can see examples from other endpoints (this is probably one of the older e2e tests)